### PR TITLE
fix broken tests

### PR DESCRIPTION
**Pull request by isinyaaa**
This attemps at fixing the broken pipe error that occurs in the project's CI [1], the broken local config manager test and also `tput`'s behavior in a non-configured environment [2].

Attempted fixes:
[1] - _https://superuser.com/questions/554855/how-can-i-fix-a-broken-pipe-error_
[2] - _https://stackoverflow.com/questions/48536791/error-tput-no-value-for-term-and-no-t-specified_

Fixes: #484, #499, #517.

### CODE REVIEW

File: tests/build_test.sh [hunk 1]
```diff
@@ -68,7 +68,7 @@ function test_kernel_build()
   ret="$?"
   assertEquals "($LINENO)" "$ret" "22"
 
-  output=$(kernel_build 'TEST_MODE' | head -1) # Remove statistics output
+  output=$(kernel_build 'TEST_MODE' | tail -n +1 | head -1) # Remove statistics output
```
Comment (kwy95) [inline tests/build_test.sh:71]: Please add a commit message.

File: src/diff.sh [hunk 2]
```diff
@@ -106,7 +106,8 @@ function diff_side_by_side()
   local diff_cmd
   local columns
 
-  columns=$(tput cols)
+  [[ "$TERM" == '' || "$TERM" == 'dumb' ]] && TPUTTERM=' -T xterm-256color'
```
Comment (kwy95) [inline src/diff.sh:109]: Please add a commit message.

**Review by kwy95 — CHANGES_REQUESTED** [review]
Thanks a lot for this patch, the unit tests output is much cleaner now. The only weird output we still get on our CI is from statistics not having permissions when trying to create files during `build_test`.
Please just add a small descriptive commit message to your commits apart from `Fixes #XYZ`.

**Review by rodrigosiqueira — CHANGES_REQUESTED** [review]
Hi,
I just have a few comments.

**Review by kwy95 — APPROVED** [review]
LGTM

**Comment by rodrigosiqueira** [comment]
Thanks a lot!
Merged to the unstable branch.

**Pull request closed without merging**